ndcube/asdf/resources/schemas/ndcube-1.0.0.yaml

```diff
@@ -0,0 +1,44 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://sunpy.org/ndcube/schemas/NDCube-1.0.0"
+
+title:
+  Represents the ndcube NDCube object
+
+description:
+  Represents the ndcube NDCube object
+
+type: object
+properties:
+  data:
+    description: "Must be compatible with ASDF serialization/deserialization and supported by NDCube."
+  wcs:
+    anyOf:
+      - tag: "tag:stsci.edu:gwcs/wcs-*"
+      - tag: "tag:astropy.org:astropy/wcs/wcs-*"
+      - tag: "tag:astropy.org:astropy/wcs/slicedwcs-*"
+      - tag: "tag:sunpy.org:ndcube/resampledwcs-*"
+      - tag: "tag:sunpy.org:ndcube/reorderedwcs-*"
+      - tag: "tag:sunpy.org:ndcube/compoundwcs-*"
+  extra_coords:
+    tag: "tag:sunpy.org:ndcube/extra_coords/extra_coords/extracoords-1.*"
+  global_coords:
+    tag: "tag:sunpy.org:ndcube/global_coords/globalcoords-1.*"
+  meta:
+    type: object
+  mask:
+    anyOf:
+      - tag: "tag:stsci.edu:asdf/core/ndarray-1.*"
+      - type: object
+      - type: boolean
```

Comment:
@DanRyanIrish Is having a single boolean as a mask something that's actually logical? We have some test fixtures which do this so I added this here.